app/templates/components/uk_components/header/_header.scss

```
@@ -269,8 +263,7 @@
       background: transparent;
     }
 
-    // Hide the inverted crown when printing in browsers that don't support SVG.
-    .govuk-header__logotype-crown-fallback-image {
+    .govuk-header__logotype-fallback-image {
```

[suggestion] I think that's a useful comment to keep around, though edited to remove the crown reference. Maybe: 

```suggestion
    // Hide the inverted logo when printing in browsers that don't support SVG.
    .govuk-header__logotype-fallback-image {
```